resource/closecaption_german.txt

```diff
@@ -870,97 +870,97 @@
 "[english]crash.RequestSealDoor2"		"<clr:238,180,34>Crash: Secure the door."
-"crash.RequestCutDoor0"		"<clr:238,180,34>Crash: Schneide es auf!"
+"crash.RequestCutDoor0"		"<clr:238,180,34>Crash: Schneidet die Tür auf!"
 "[english]crash.RequestCutDoor0"		"<clr:238,180,34>Crash: Cut it open!"
-"crash.RequestCutDoor1"		"<clr:238,180,34>Crash: Worauf wartest du noch? Mach es auf!"
+"crash.RequestCutDoor1"		"<clr:238,180,34>Crash: Worauf wartet ihr noch? Macht sie auf!"
 "[english]crash.RequestCutDoor1"		"<clr:238,180,34>Crash: What're you waiting for? Get it open!"
-"crash.RequestShootDoor0"		"<clr:238,180,34>Crash: Oh ja, lasst uns diese Tür in die Hölle sprengen!"
+"crash.RequestShootDoor0"		"<clr:238,180,34>Crash: Au ja, lasst uns diese Tür in die Luft jagen!"
 "[english]crash.RequestShootDoor0"		"<clr:238,180,34>Crash: Oh yeah, let's blast this door to hell!"
-"crash.RequestShootDoor1"		"<clr:238,180,34>Crash: Sprengt es auf!"
+"crash.RequestShootDoor1"		"<clr:238,180,34>Crash: Sprengt sie auf!"
 "[english]crash.RequestShootDoor1"		"<clr:238,180,34>Crash: Blast it open!"
-"crash.RequestShootDoor2"		"<clr:238,180,34>Crash: Schießt es auf!"
+"crash.RequestShootDoor2"		"<clr:238,180,34>Crash: Brecht sie auf!"
 "[english]crash.RequestShootDoor2"		"<clr:238,180,34>Crash: Shoot it open!"
-"crash.CuttingDoor0"		"<clr:238,180,34>Crash: Crash, cutin' door."
+"crash.CuttingDoor0"		"<clr:238,180,34>Crash: Crash, schneide mich durch die Tür."
 "[english]crash.CuttingDoor0"		"<clr:238,180,34>Crash: Crash, cuttin door."
-"crash.CuttingDoor1"		"<clr:238,180,34>Crash: Verstanden. Ich schneide die Tür auf."
+"crash.CuttingDoor1"		"<clr:238,180,34>Crash: Verstanden. Schneide mich durch die Tür."
 "[english]crash.CuttingDoor1"		"<clr:238,180,34>Crash: Roger that. Cuttin door."
-"crash.SealingDoor0"		"<clr:238,180,34>Crash: Crash, Dichtungstür."
+"crash.SealingDoor0"		"<clr:238,180,34>Crash: Crash, versiegle Tür."
 "[english]crash.SealingDoor0"		"<clr:238,180,34>Crash: Crash, sealing door."
-"crash.SealingDoor1"		"<clr:238,180,34>Crash: Schließt die Tür gut und fest ab."
+"crash.SealingDoor1"		"<clr:238,180,34>Crash: Tür wird versiegelt und verplombt."
 "[english]crash.SealingDoor1"		"<clr:238,180,34>Crash: Sealing door nice and tight."
-"crash.Biomass0"		"<clr:238,180,34>Crash: Schwarm-Biomasse - verbrennt alles!"
+"crash.Biomass0"		"<clr:238,180,34>Crash: Schwarm-Biomasse – brennt sie nieder!"
 "[english]crash.Biomass0"		"<clr:238,180,34>Crash: Swarm biomass--burn it all!"
-"crash.Biomass1"		"<clr:238,180,34>Crash: Biomasse? Mann, ich hasse diesen Scheiß."
+"crash.Biomass1"		"<clr:238,180,34>Crash: Biomasse? Mann, wie ich diesen Scheiß hasse."
 "[english]crash.Biomass1"		"<clr:238,180,34>Crash: Biomass? Man, I hate this shit."
-"crash.Biomass2"		"<clr:238,180,34>Crash: Biokram. Was zum Teufel ist das überhaupt?"
+"crash.Biomass2"		"<clr:238,180,34>Crash: Biomüll. Was zum Teufel ist das überhaupt für 'n Zeug?"
```

Comment:
Ja, doch, klingt gut. Er nennt es in jeder Zeile anders, da passt Biomüll eigentlich ganz gut :)